doc/nrf-bm/links.txt

```diff
@@ -11,6 +11,7 @@
 .. ### Source: docs.nordicsemi
 
 .. _`nRF54L15 DK`: https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf54l15dk/doc/index.html#nrf54l15dk-nrf54l15
+.. _`0.9.3`: https://docs.nordicsemi.com/bundle/comp_matrix_nrf54l15/page/COMP/nrf54l15/nrf54l15_ic_rev_comp_with_dev_hw.html
```

Comment:
Can it be changed to a better (more unique) tag name for this? E.g. `Revision 0.9.3`?

Reply:
Good point, although I think 'revision' refers to SoCs while DKs have versions. Changed to '`nRF54L15 DK v0.9.3`'